src/components/Chat.js

```diff
@@ -100,6 +100,7 @@ export const Chat = ({
       minInputToolbarHeight={normalize(96)}
       placeholder={placeholder}
       scrollToBottom
+      scrollToBottomComponent={() => <Icon.ArrowDown />}
```

Comment:
does it need to be a function or would `Icon.ArrowDown` also work?

Reply:
This will unfortunately cause the application to crash. A function that will generate this prop return is needed.

image.png